lighthouse-core/lib/proto-preprocessor.js

```diff
@@ -56,9 +56,10 @@ function processForProto(result, opts = {}) {
           audit.scoreDisplayMode = 'notApplicable';
         }
       }
-      // Drop raw values. https://github.com/GoogleChrome/lighthouse/issues/6199
-      if (!keepRawValues && 'rawValue' in audit) {
-        delete audit.rawValue;
+      // Drop numeric values until we decide what to do with the optional type in proto.
```

Comment:
punting on the question of `DoubleValue`/`Value`/whatever in proto by maintaining the deleting by default behavior of `rawValue`. We can change it to whatever later.

Reply:
I think it is going to have to be `Value` but this is something that we can punt 🏈

Reply:
On the other hand.. this seems a great time to not punt on the proto typing for this prop. :) All our PSI API users currently don't get rawValue because we're punting and thus have to parse strings like "3,241ms" in order to get some metric values.

`.numericValue` being typed as the mega-generic proto `Value` seems real bad. 

What are the challenges to typing `.numericValue` as `DoubleValue`?

Reply:
Maybe I misunderstood but we are punting on adding the property at all to the proto right, not proposing to start with it as `Value`?

Reply:
Afaik DoubleValue doesn't support a difference between 'null' or 'undefined' and '0' since it is the default.  I can do some more testing around that, but thats the crux.  BUT if we don't care about undefined = 0, then we can for sure use a more typical and better proto type!

Reply:
Hold up - DoubleValue _is_ nullable. `double` and `float` are the ones that can't distinguish between 0 and not provided.

Reply:
> Maybe I misunderstood but we are punting on adding the property at all to the proto right, not proposing to start with it as Value?

yeah, that's what I was proposing, not changing the status quo on proto, but we can do this for real

Reply:
@exterkamp and I just tested a few possibilities and, yeah, `DoubleValue` will work well for us. `0` comes through, and then any audit with it not set will come back with no `numericValue` in JSON, which works just fine for us.